### ISSUE TITLE
Error and solution with rqt launch

### ISSUE THREAD

**Issue by Betancourt20** [Kinovarobotics/kinova-ros#359]
## Description

Problem with rqt gui
## Version
ROS distribution :  Ubunutu 20.04 ROS noetic
## Problem 

Traceback (most recent call last):
  File "/opt/ros/noetic/lib/rqt_gui/rqt_gui", line 13, in <module>
    sys.exit(main.main())
  File "/opt/ros/noetic/lib/python3/dist-packages/rqt_gui/main.py", line 61, in main
    return super(
  File "/opt/ros/noetic/lib/python3/dist-packages/qt_gui/main.py", line 614, in main
    perspective_manager.import_perspective_from_file(
  File "/opt/ros/noetic/lib/python3/dist-packages/qt_gui/perspective_manager.py", line 360, in import_perspective_from_file
    self._convert_values(data, self._import_value)
  File "/opt/ros/noetic/lib/python3/dist-packages/qt_gui/perspective_manager.py", line 422, in _convert_values
    self._convert_values(groups[group], convert_function)
  File "/opt/ros/noetic/lib/python3/dist-packages/qt_gui/perspective_manager.py", line 419, in _convert_values
    keys[key] = convert_function(keys[key])
  File "/opt/ros/noetic/lib/python3/dist-packages/qt_gui/perspective_manager.py", line 429, in _import_value
    return QByteArray.fromHex(eval(value['repr(QByteArray.hex)']))
  File "<string>", line 1, in <module>
TypeError: arguments did not match any overloaded call:
  QByteArray(): too many arguments
  QByteArray(int, str): argument 1 has unexpected type 'str'
  QByteArray(Union[QByteArray, bytes, bytearray]): argument 1 has unexpected type 'str'
[kinova_rqt-2] process has died [pid 22822, exit code 1, cmd /opt/ros/noetic/lib/rqt_gui/rqt_gui --perspective-file /home/unknown/ws_moveit/src/kinova-ros/kinova_control/launch/j2n6s300.perspective __name:=kinova_rqt __log:=/home/unknown/.ros/log/a21166a2-356b-11ec-99cd-cd482ed4316d/kinova_rqt-2.log].
log file: /home/unknown/.ros/log/a21166a2-356b-11ec-99cd-cd482ed4316d/kinova_rqt-2*.log
^[[A^C[rosout-1] killing on exit
[master] killing on exit
shutting down processing monitor...
... shutting down processing monitor complete
done

## Solution
In turns out that things changed for noetic. So, I solved the problem based on 
https://answers.ros.org/question/365220/rqt_gui-error-in-perspective-manager-in-ros-noetic/
Basically the problems comes from the launch file, i.e., /kinova_control/launch/kinova_rqt  and this file calls j2n6s300.perspective . Then we need to change the code lines 50 and 54. I.e. after replacing the value with only byte content, like b'000000ff' and removing QtCore.QByteArray() creation, the rqt_gui is able to start without errors. 

Hope this helps!

**Comment by felixmaisonneuve** [comment]
Hi @Betancourt20,

I appreciate your feedback very much. 

However, I already did this modification in PR #340. I suppose you are using the melodic-devel branch on ROS noetic. I would suggest to change to our noetic-devel branch. 

Thank you nonetheless for using our issue template and giving a clear solution to the problem, it helps a lot.

Best,
Felix
